src/Constants.ts

```diff
@@ -0,0 +1 @@
+export const APP_NAME = 'lfp-appeals-frontend';
```

Comment:
Are we planning on using this Constants.ts in the future? Worried it might lead to some bad practices in the future where we just dump consts that we dont know where to put in it.  Could we maybe move this constant to another file or perhaps name this one specifically for the APP_NAME?

Reply:
I have no defined plan for it right now, partially cause I do not know what the future is. 

Constants file was created cause it is better place than ConfigLoader. That is it.

Reply:
Alternatively it can be in `App.ts` or `Server.ts`